merryface/pain_tracker
# Component today score

# DayScore

DayScore is made of three components:

- DayScore
- TreatmentPill
- DayScoreCard

DayScoreCard contains the other two components and outputs them on the page. 

**Reason for calling it Score and not Rating:** I figured we can differentiate the individual scales and the page's total. Score would be the average of that page's ratings. Please let me know if you have a better idea.

Score takes average of the values in the `ratings` object and displays it to 1dp
Treatment pill takes in a colour and a boolean found in a `treatments` object from state.

## To test
- Go to the homepage of the app at localhost:3000

_Score_
- Verify that there is a today's score output at the top of the pag 
- Click or tap on the rating buttons
- Verify that the score outputs the sum of all selected values divided by 5 (no redux state value)

_Treatment Pills_
- Verify that there are three pill buttons stacked vertically to the right of the Score. They should be green red and blue (placehold colours)
- Go into `data/initial.ts` and change values in `treatments` to `false` from `true` and check that the pills turn grey
